### PR TITLE
Make this bundle compatible with Symfony3

### PR DESCRIPTION
**Pull request by benji07**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Thanks. Could you please add related tests?

**Comment by benji07** [comment]
I don't really know how to test that, you don't have test on the `SonataIntlExtension` file and none of the LocaleDetector have test

**Comment by benji07** [comment]
@Soullivaneuh done (for the locale detector)

**Comment by Soullivaneuh** [comment]
Well, a test for `SonataIntlExtension` would be great too IMHO, with each SF version test.

**Comment by benji07** [comment]
@Soullivaneuh done

File: Resources/config/intl.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
     <parameters>
         <parameter key="sonata.intl.locale_detector.request.class">Sonata\IntlBundle\Locale\RequestDetector</parameter>
+        <parameter key="sonata.intl.locale_detector.request_stack.class">Sonata\IntlBundle\Locale\RequestStackDetector</parameter>
```
Comment (Soullivaneuh) [inline Resources/config/intl.xml]: This is not recommended by SF best practices anymore. Please use class names directly.

Comment (benji07) [inline Resources/config/intl.xml]: i did that to remain consistent with the other services declared here

Comment (Soullivaneuh) [inline Resources/config/intl.xml]: I understand, but removing this is already a BC break so no need to had some hard work here. :-)

**Comment by benji07** [comment]
@Soullivaneuh i change the service file.

**Comment by Soullivaneuh** [comment]
ping @sonata-project/contributors for a second look.

**Comment by OskarStark** [comment]
What about addind symfony 3.0 to travis?

**Comment by benji07** [comment]
@OskarStark https://github.com/sonata-project/SonataIntlBundle/blob/master/.travis.yml#L38

**Comment by benji07** [comment]
ping @Soullivaneuh any news ?

**Comment by OskarStark** [comment]
you are right @benji07 

looks good to me.

@rande what do you think?

**Comment by OskarStark** [comment]
@benji07 please squash your commits before we can merge this PR!

Thank you

**Comment by Soullivaneuh** [comment]
Do not use `version_compare`.

**Comment by Soullivaneuh** [comment]
You can have `symfony/http-kernel` 2.3 and `symfony/dependency-injection` 2.5 for example, and your method will not work.

**Comment by benji07** [comment]
@Soullivaneuh i remove every call to version_compare

**Comment by benji07** [comment]
@OskarStark i rebase my PR

**Comment by Soullivaneuh** [comment]
:+1:

**Comment by OskarStark** [comment]
Thank you for this great work! :+1:

**Pull request merged**